tests/benchmarks/vecsim-arxiv-titles-384-angular-filters-m16-ef-128-fulltext-filter.yml

```diff
@@ -2,15 +2,10 @@ version: 0.5
 name: "vecsim-arxiv-titles-384-angular-filters-m16-ef-128-fulltext-filter"
 metadata:
   component: "vecsim"
-remote:
- - type: oss-standalone
- - setup: redisearch-m5
+
 timeout_seconds: 1800
 setups:
   - oss-standalone
-  - oss-cluster-03-primaries
-  - oss-cluster-05-primaries
-  - oss-cluster-09-primaries
```

Comment:
Does that mean we no longer run the vecsim benchmarks on cluster mode?

Reply:
@GuyAv46 i've removed the cluster envs from those benchmarks (the ones loading from rdb) given if you load from rdb the cluster benchmark wont run. This means we have 4 benchmarks for vecsim cluster (based on ann) but this PR is not reducing/increasing it. 
I'll work on adding more cluster benchmarks that are vecsim related